renku/ui/service/serializers/templates.py

```diff
@@ -108,6 +108,7 @@ class ManifestTemplateSchema(Schema):
     icon = fields.String(
         load_default=None, metadata={"description": "base64 encoded icon for the template in PNG format"}
     )
+    ssh_supported = fields.Boolean(load_default=False, metadata={"description": "Whether the template supports SSH"})
```

Comment:
This is properly supported in the SSH feature branch (It requires a new API version of the endpoint since we changed the schema)

Reply:
I've noticed an integration test if failing because of this. I'll revert it.

Reply:
yes it's failing because the property is already in the templates repo in `master` but the SSH changes haven't been merged yet in renku-python. But you can ignore that failure in this PR